py/pysparkling/context.py

```diff
@@ -119,12 +119,17 @@ def getOrCreate(spark_context, conf = None):
         # Create H2O REST API client
         h2o.connect(ip=h2o_context._client_ip, port=h2o_context._client_port)
         h2o_context.is_initialized = True
+        # Stop h2o when running standalone pysparkling scripts and the user does not explicitly close h2o
+        atexit.register(lambda: h2o_context.stop())
         return h2o_context
 
     def stop(self):
         warnings.warn("H2OContext stopping is not yet fully supported...")
         self._jhc.stop(False)
 
+    def __del__(self):
+        self.stop()
```

Comment:
One more comment: if somebody will do:
```
def fu():
   h2oContext = ...
   ...
   h2oContext.stop()

func()
```

Then `stop` will be called twice - one explicitly by user, the second time by `__del__` - is it a problem?